postcss.config.js

```diff
@@ -28,6 +28,10 @@ module.exports = ({ file }) => {
         /data-v-.*/,
         /^ec-/,
       ],
+      whitelistPatternsChildren: [
+        /^ec-/,
+        /^ebo-/,
```

Comment:
We have to do this in ebo repo. remove the ebo- bit from here. chameleon has no clue about EBO.